### DISCUSSION TITLE
Binaries?

### DISCUSSION THREAD

**Discussion by the-robo** [FormerLurker/ArcWelderLib#38] [Q&A] [answered]
Are there any binaries for Windows users?  I don't have a compiler.  If not, could they be posted please?

**Comment by FormerLurker** [comment, marked as answer]
[Latest releases here](https://github.com/FormerLurker/ArcWelderLib/releases/tag/1.1.0)
[Latest development release as of today here](https://github.com/FormerLurker/ArcWelderLib/actions/runs/538259176) (note that the macos build doesn't work here, some problem with the github actions).

**Comment by FormerLurker** [comment]
Please mark this as the solution if it helped you out.  Thanks!

**Comment by FormerLurker** [comment]
FYI, a working macos build is now up!
